assets/js/shims.js

```diff
@@ -0,0 +1,5 @@
+export let process = {
```

Comment:
Is this addition required for the Algolia search?

Reply:
Yeah. Its because they have some code that thinks its running on Node and not in a browser. This just mocks out the `process` global so that code can load and then ultimately do nothing.